src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
```

Comment:
Typically I wouldn't use the id attribute in a single page app. There's really no guarantee that ids are unique and if they get reused/duplicated by someone else in another component on the same page it can sometimes cause weird side effects. This is a small app, so it's probably never going to run into collisions. So just wondering, is there a specific reason for using the id attribute rather than a class attribute?